motionplan/armplanning/cBiRRT.go

```diff
@@ -482,11 +482,22 @@ func (mp *cBiRRTMotionPlanner) getFrameSteps(percentTotalMovement float64, doubl
 			jRange := math.Abs(u - l)
 			pos[i] = jRange * percentTotalMovement
 
-			if isMoving && double {
-				pos[i] *= 2
-			}
-			if !isMoving {
-				pos[i] *= .25 // we move non-moving frames just a little if we have to get them out of the way
+			if isMoving {
+				if iterationNumber > 20 {
+					pos[i] *= 2
```

Comment:
So I'm guessing that something else is choosing which "direction" to move each joint and `pos` is just to inform the "magnitude" of how aggressive we should be trying to move it?

Assuming `percentTotalMovement` is [0, 1], we've scaled `pos[i]` to be between limits. This doubling can presumably take `pos[i]` to be out of its intended bounds? Do we need to claw the value back fit within the limits?

Sorry for commenting on the commit and not the PR itself.

Reply:
actually, it's correct :) 
sorry.
percent is .01 right now, so i think it's probably fine

this whole system is a mess